tests/ballerina-integration-test/src/test/java/org/ballerinalang/test/util/TestUtils.java

```diff
@@ -164,4 +171,17 @@ public static void prepareBalo(Object test) throws URISyntaxException {
         Path target = Paths.get(path).getParent();
         System.setProperty(BLangConstants.BALLERINA_HOME, target.toString());
     }
+
+    public static KeyStore getKeyStore(File keyStore)
+            throws IOException, KeyStoreException, CertificateException, NoSuchAlgorithmException {
+        KeyStore ks = null;
+        String tlsStoreType = "PKCS12";
+        if (keyStore != null) {
```

Comment:
Shall we first check if the 'keyStore == null' and then return it? IMO it will make the code look cleaner